### PR TITLE
Update appveyor.yml to use external script

### PR DESCRIPTION
**Pull request by qinezh**
@markcowl

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -2,61 +2,30 @@ branches:
...
+  - cd ..
+build_script:
+  - node azpsci/out/ci0.js src c:\projects\_output
+
+test: off
+on_success:
   - git clone -q --branch=%target_branch% %content_repo% %TEMP%\Azure
   - cd %TEMP%\Azure
-  - ps: ls $global:output -dir | % { copy $_.FullName (ls -dir | select -First 1) -Recurse -Force }
+  - ps: ls c:\projects\_output -dir | % { copy $_.FullName (ls -dir | select -First 1) -Recurse -Force }
```
Comment (twitchax) [inline appveyor.yml]: Why change this away from a build variable?

Comment (markcowl) [inline appveyor.yml]: Cannot assume this will be on c: drive

File: appveyor.yml [hunk 2]
```diff
@@ -2,61 +2,30 @@ branches:
...
-          {
-            robocopy $help "*.md" (Join-Path $global:output $folder | Join-Path -ChildPath (gi $psd1).BaseName | Join-Path -ChildPath ("v" + $matches[0]))
-          }
-        }
-      }
-test: off
-shallow_clone: true
-on_success:
+before_build:
+  - set path=%programfiles(x86)%\Microsoft SDKs\TypeScript\2.2;%path%
```
Comment (markcowl) [inline appveyor.yml]: This is making all kinds of assumptions about the environment it is running in.  What are the required environment variables, and why is it assumign it is runnign on a 64-bit system?

File: appveyor.yml [hunk 3]
```diff
@@ -2,61 +2,30 @@ branches:
...
+  - ps: Install-Product node $env:nodejs_version
   - git config --global credential.helper store
   - ps: ac "$env:USERPROFILE\.git-credentials" "https://$($env:github_access_token):x-oauth-basic@github.com`n"
   - git config --global user.email %email%
   - git config --global user.name %name%
+  - git clone -q https://github.com/MicrosoftDocs/azure-psdocs-ciscripts azpsci
+  - cd azpsci
+  - npm install
+  - npm install -g typescript
```
Comment (markcowl) [inline appveyor.yml]: Why install this globally?

File: appveyor.yml (hunk 1 above)
Comment (markcowl) [inline appveyor.yml]: This cannot assume it's working on the c drive

**Comment by qinezh** [comment]
There's an assumption, the script is for AppVeyor. Based on the assumption, the script is running on the c drive according to https://www.appveyor.com/docs/appveyor-yml.

File: appveyor.yml (hunk 1 above)
Comment (qinezh) [inline appveyor.yml]: @markcowl  I think it works even if the script not working on the c drive.

Comment (qinezh) [inline appveyor.yml]: @twitchax Beacuse it doesn't need the variable anymore.

**Pull request merged**